clang/test/dpct/replace-align.cu

```diff
@@ -1,6 +1,9 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
 // RUN: dpct --format-range=none -out-root %T/replace-align %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
 // RUN: FileCheck %s --match-full-lines --input-file %T/replace-align/replace-align.dp.cpp
 
+#include <cuda_fp8.h>
```

Comment:
cuda_fp8.h is only added in cuda-11.8 or later, so line 1 and line 2 also needs to be adjusted.